main/ertp/ERTP-Guide/Overview.md

```diff
@@ -0,0 +1,260 @@
+  The description of an asset, it is not an asset itself, having no economic scarcity or economic value.
+  Made up of a:
+  - *[value](./Amounts.md)* *[(Glossary)](https://agoric.com/documentation/glossary/#value)*:  The
+  size of the amount.
+  - *[brand](./Amounts.md)* *[(Glossary)](https://agoric.com/documentation/glossary/#brand)* The
+  kind of asset. 
+  
+  You can think of these as the "how many" and the "what" of an asset, for example, 7 quatloos (an imaginary
+  currency).
+- **[AmountMath](./AmountMath.md)** **[(Glossary)](https://agoric.com/documentation/glossary/#amountmath)**:
+Methods to do math operations on an `amount`..
+- **[Purse](./PursesAndPayments.md)** **[(Glossary)](https://agoric.com/documentation/glossary/#purse)**: an
+  object for long term digital assets storage. 
+- **[Payment](./PuresAndPayments.md)** **[(Glossary)](https://agoric.com/documentation/glossary/#payment)**:
+  an object for transfering digital assets to another party.
+- **[Mint](./IssuersAndMints.md)** **[(Glossary)](https://agoric.com/documentation/glossary/#mint)**: an
+  object that creates new `payment` objects of a specific `brand`. Each `brand` has
+  a one to one relationship with a `mint` and vice versa.
+- **[Issuer](./IssuersAndMints.md)** **[(Glossary)](https://agoric.com/documentation/glossary/#issuer)**: 
+an object linked to a single `mint` object that creates empty `purse` objects
+for that mint's `brand` and operates on any `payment` of that `brand`. 
+
+## Fungible and Non-Fungible Assets
+
+There are two kinds of assets,
+*[fungible](https://agoric.com/documentation/glossary/#fungible)* and
+*[non-fungible](https://agoric.com/documentation/glossary/#non-fungible)*. 
+
+Fungible assets are interchangeable. For example, if you have 100
+one-dollar bills and need to pay someone 5 dollars, it doesn't matter
+which five of your one-dollar bills you give them. 
+
+Non-fungible assets are of the same type, but are not interchangeable
+and specific items must be used. For example, you might have 100
+tickets. But someone wanting to buy even a General Admission ticket from you will want one
+for a specific date and time. This might also affect the price; you'll want to charge more
+for a Friday evening ticket than a Wednesday matinee ticket, even if it's for the same show.
+
+## Amounts are not assets
+
+**IMPORTANT**: Despite how it may seem, an `amount` is only a description of an asset, not
+an asset in and of itself. Actual assets are in `purse` and `payment` objects, but the
+assets aren't ever embodied in any particular JavaScript object. Instead, what makes the 
+assets exist is the issuer having a mapping from the `purse` or `payment` to an `amount` (description).
+There is no `amount` object stored in a `purse` or `payment`.
+
+Some analogies may be helpful in understanding this:
+
+- Let's say Caltech replicates its [famous hack of the Rose Bowl scoreboard](https://www.admissions.caltech.edu/pranks).
```

Comment:
I think these examples do not get at the right point - changing the amount balance in the records held by the issuer would actually change the digital assets. It's whether the record is the source of truth or not that actually matters.